lib/appsignal/instrumentation.ex

```diff
@@ -38,8 +38,11 @@ defmodule Appsignal.Instrumentation do
 
   def instrument(name, category, fun) do
     instrument(fn span ->
-      @span.set_name(span, name)
-      @span.set_attribute(span, "appsignal:category", category)
+      _ =
```

Comment:
dialyzer wants you to assign the value to a variable even though it's unused?

Reply:
Dialyzer's has an optional option named unmatched returns, which prints a warning for all returning functions that are not caught. These might possibly cause issues, as not handling these results could result into unexpected outcomes. In our case, we know that our Span functions either return Spans or nils, and that all functions in the Span module handle these correctly.

We use the `_` to silence Dialyzer, essentially "tricking" it into thinking we are assigning it to a variable, while we're not.